docs/src/modules/components/MarkdownElement.js

```diff
@@ -380,31 +380,31 @@ const Root = styled('div')(
       fontSize: 10,
       '&:hover': {
         '& .MuiCode-copy': {
-          display: 'block',
+          display: 'inline-flex',
+          flexDirection: 'row-reverse',
```

Comment:
What about puttin the text before the icon, and then don't use either the `width` and `flexDirection` properties

```jsx
<span className="MuiCode-copyKeypress">
  <span>(or</span> {key}C<span>)</span>
</span>
{/* material-ui/no-hardcoded-labels */}
  {isCopied ? (
    <LibraryAddCheckRoundedIcon sx={{ fontSize: 18 }} />
  ) : (
    <ContentCopyRoundedIcon sx={{ fontSize: 18 }} />
  )}
  ```

Reply:
Uhm, tried locally and it wasn't working 🤔 The "Or..." label was getting on top of the button itself.